CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
    - Header file `include/secp256k1_ellswift.h` which defines the new API.
    - Document `doc/ellswift.md` which explains the mathematical background of the scheme.
    - The [paper](https://eprint.iacr.org/2022/759) on which the scheme is based.
+ - We now test the library with unreleased development snapshots of GCC and Clang. This gives us an early chance to catch miscompilations and constant-time issues introduced by the compiler (such as these that led to the previous two releases).
+
+#### Fixed
+ - Certain internal library symbols are no longer exported on Windows DLL builds.
```

Comment:
Maybe drop the word "Certain"?

I'm not a native English speaker, but the sentence in its current variant makes me think that there are still internal symbols that are exported.

Reply:
rephrased